### PR TITLE
Add ITA / ITTS examples / documentation

### PR DESCRIPTION
**Pull request by fidencio**
Intel Tiber Trust Services (ITTS), formerly known as Intel Trust Authority (ITA), is Intel's own Zero Trust Attestation Service.

Its support has already been added into the Confidential Containers Trustee project, is already being tested as part of Kata Containers CI for TDX, and now is being brough up to the Confidential Containers Trustee Operator project, so it can be easily used by whoever wishes to do so.

### CODE REVIEW

File: docs/ita.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+openssl genpkey -algorithm ed25519 > privateKey
+openssl pkey -in privateKey -pubout -out publicKey
+oc create secret generic kbs-auth-public-key --from-file=publicKey -n trustee-operator-system
+```
+
+### Trustee ConfigMap object
+
+This command will create the ConfigMap object that provides Trustee all the needed configuration.
+
+Note: you'd probably need to change the Intel API key. Now it is set to `api_key": "tBfd5kKX2x9ahbodKV1..."`
```
Comment (bpradipt) [inline docs/ita.md:51]: Adding a link to create an API key will be helpful imho. I'm assuming using one's own API key is must for ITA.

Comment (fidencio) [reply docs/ita.md:54]: I've put a link to ITA / ITTS at the beginning of the document,

File: config/samples/all-in-one/ita-kbs-config.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kbs-config
```
Comment (lmilleri) [inline config/samples/all-in-one/ita-kbs-config.yaml:4]: A ConfigMap with the same name already exists in the directory. This causes problems when running commands like `kustomize build .`.
The same applies for the resource-policy.
What about changing the object name for ita specific resources?
Please check the microservices folder as well.

File: docs/ita.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+  labels:
+    app.kubernetes.io/name: kbsconfig
+    app.kubernetes.io/instance: kbsconfig
+    app.kubernetes.io/part-of: trustee-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: trustee-operator
+  name: kbsconfig
+  namespace: trustee-operator-system
+spec:
+  kbsConfigMapName: kbs-config-cm
```
Comment (lmilleri) [inline docs/ita.md:168]: Shouldn't it be kbs-config?

**Review by lmilleri — CHANGES_REQUESTED** [review]

File: config/samples/all-in-one/ita-kbs-config.yaml (hunk 2 above)
Comment (fidencio) [reply config/samples/all-in-one/ita-kbs-config.yaml]: I've changed the name to ita-kbs-config, and did the same for the resource-policy.

File: docs/ita.md (hunk 3 above)
Comment (fidencio) [reply docs/ita.md]: And I reflected the changes here as well, including this one that you pointed out.

**Review by lmilleri — APPROVED** [review]

**Pull request merged**